sentinel-extension/sentinel-parameter-flow-control/src/main/java/com/alibaba/csp/sentinel/slots/block/flow/param/ParamFlowRuleUtil.java

```diff
@@ -168,7 +163,7 @@ public static void fillExceptionFlowItems(ParamFlowRule rule) {
 
             if (flowRules == null) {
                 // Use hash set here to remove duplicate rules.
-                flowRules = new HashSet<>();
+                flowRules = new LinkedHashSet<>();
```

Comment:
Is this change necessary?

Reply:
To keep rules in order previously, has been reverted.